src/panels/KeybindsPanel/FullKeyboard/FullKeyboard.tsx

```diff
@@ -51,19 +51,30 @@ export function FullKeyboard({
     }
   }
 
-  function getActionsForKey(key: string): Action[] {
-    // Find all action identifiers matching the given key and current modifiers
+  function getActionsForKey(key: string): KeybindInfoType[] {
+    // Find all action identifiers matching the given key and current modifiers.
+    // If no modifiers are active, it will match all keybinds with the given key.
```

Comment:
```suggestion
    // Find all action identifiers matching the given key and current modifiers
    // If no modifiers are active, it will match all keybinds with the given key
```